### ISSUE TITLE
[BUG] Fuel decreasing on stand with engines and APU off

### ISSUE THREAD

**Issue by claguazza** [flybywiresim/a32nx#2558]
**Mod Version**

Latest Stable

**Describe the bug**

While on stand with less than 3000kgs of FOB, the fuel starts decreasing quite quickly and message "OUTR TK FUEL XFRD" appears steady

**To Reproduce**

1. Set up a flight from stand, cold&dark at any airport
2. Start batteries and external power
3. Load via MCDU a quantity of fuel less than 3000kgs

**Expected behavior**

Fuel should not decrease having both engines and APU off

**Actual behavior**

Message "OUTR TK FUEL XFRD" appears steady and fuel on board (FOB) starts decreasing quickly

**References**

**Additional context**

The same message "OUTR TK FUEL XFRD" appears flashing inflight when FOB drops below 3000kgs but in this case the fuel reduces normally as per engines running
Was this working before/when did the issue start occurring?
It happened from the first time I installed the latest stable version 0.5.0
Is this a problem in the vanilla unmodded game?
Not tested

Discord username (if different from GitHub):

**Comment by Watsi01** [comment]
Just as an info: "OUTR TK FUEL XFRD" tells you, that the fuel in your inner wing tanks dropped below 750kg (each). Then fuel from the outer wing tanks will automatically be pumped to the inner tanks.

**Comment by claguazza** [comment]
Thanks for your info. I was familiar with that message since I have flown the 320 in real life. Good to share that with everybody. What is weird is the behaviour on ground before starting engines and apu. Hopefully it can be fixed somehow

**Comment by MisterChocker** [comment]
duplicate of #2083
